### PR TITLE
Sensor error reporting improvements

### PR DESCRIPTION
**Pull request by Kami**
Include a traceback when sensor.run throws and also log the same message.

### CODE REVIEW

File: st2reactor/st2reactor/container/sensor_wrapper.py [hunk 1]
```diff
@@ -276,8 +277,11 @@ def run(self):
         try:
             self._sensor_instance.run()
         except Exception as e:
-            raise Exception('Sensor "%s" run method raised an exception: %s' %
-                            (self._class_name, str(e)))
+            # Include traceback
+            msg = 'Sensor "%s" run method raised an exception: %s.\nTraceback: %s'
+            msg = msg % (self._class_name, str(e), traceback.format_exc())
+            self._logger.warn(msg)
```
Comment (manasdk) [inline st2reactor/st2reactor/container/sensor_wrapper.py]: self._logger.warning(msg, exc_info=True)

This way you won't have to deal with the traceback etc.

Comment (Kami) [inline st2reactor/st2reactor/container/sensor_wrapper.py]: Heh, true, but we still need it for the exception message :)

Comment (manasdk) [inline st2reactor/st2reactor/container/sensor_wrapper.py]: Are you sure the exception message is not included? It is pretty much the same as calling

log.exception(msg) without the error level. I am quite certain log.exception includes the exception message.

Comment (Kami) [inline st2reactor/st2reactor/container/sensor_wrapper.py]: Sorry, I meant for the exception which is thrown after self._logger.warn call.

Comment (manasdk) [inline st2reactor/st2reactor/container/sensor_wrapper.py]: Then why bother creating a different exception? re-raise the original exception

Comment (manasdk) [inline st2reactor/st2reactor/container/sensor_wrapper.py]: I guess you do it to include some more information. Generally, the double traceback server minimal value especially if the exception is logged in the same file as the original. 

Anyway its done. So be it.

**Pull request merged**